lua/lsp-status/current_function.lua

```diff
@@ -18,7 +18,14 @@ local function current_function_callback(_, _, result, _, _)
 
   local function_symbols = util.filter(util.extract_symbols(result),
     function(_, v)
-      return v.kind == 'Class' or v.kind == 'Function' or v.kind == 'Method'
+      return (v.kind == 'Class' or
```

Comment:
Style nitpick: I would prefer that we define a module-local table mapping these names to `true` (e.g. `local scopes = {Function = true, Method = true, ...}`), and then here check if `v.kind` is contained, e.g. by `return scopes[v.kind]`. This makes it easier to update the set of scopes at a later date, and makes for cleaner code at only a small efficiency hit.

Reply:
Implemented this! (In the messiest commit history of my life because I thought this was trivial and I could commit directly to the PR branch, but oh well. Presumably gonna get squashed, so *shrugs*.)

Thanks for the suggestion, definitely agree with it. The modularity will also make it easier to make configurable.